README.md

```diff
@@ -170,7 +170,7 @@ $ ros2 trace stop session_name    # Stop tracing after starting or resuming
 
 Run each command with `-h` for more information.
 
-You must [install the kernel tracer](#building) if you want to enable kernel events (using the `-k`/`--kernel-events` option).
+You must [install the kernel tracer](#building) if you want to enable [kernel](https://lttng.org/docs/v2.13/#doc-tracing-the-linux-kernel) events (using the `-k`/`--kernel-events` option) or syscalls (using the `--syscalls` option).
```

Comment:
how about using `https://lttng.org/docs/#doc-tracing-the-linux-kernel` here? that is redirected to latest doc version if we do not want to specify the version from this doc.

```suggestion
You must [install the kernel tracer](#building) if you want to enable [kernel](https://lttng.org/docs/#doc-tracing-the-linux-kernel) events (using the `-k`/`--kernel-events` option) or syscalls (using the `--syscalls` option).
```

Reply:
I've thought about this before, because the READE has many links to the LTTng docs. I prefer linking to a specific version in case the docs change in the future. This way, when we switch to a new version of Ubuntu that has a new version of LTTng, I'm forced to update the links and verify that they still point to the right sections. If I don't update the links, then they still point to the right section, but not the right version. Then I think it's easier to find the corresponding section in the docs for the new version, as opposed to finding the section in the new version of the docs from just the `#fragment` (if the link doesn't point to the right section).

Reply:
I see that, thanks!